src/com_tjnotifications/admin/sql/install.mysql.utf8.sql

```diff
@@ -1,20 +1,20 @@
 CREATE TABLE IF NOT EXISTS `#__tj_notification_providers` (
-  `provider` varchar(100) NOT NULL,
-  `state` int(1) NOT NULL,
+  `provider` varchar(100) NOT NULL DEFAULT '',
+  `state` int(1) NOT NULL DEFAULT '0',
    primary key(provider)
  ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
 
 CREATE TABLE IF NOT EXISTS `#__tj_notification_templates` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `key` varchar(100) NOT NULL,
-  `client` varchar(100) NOT NULL,
-  `title` varchar(100) NOT NULL,
-  `state` int(11) NOT NULL,
+  `key` varchar(100) NOT NULL DEFAULT '',
+  `client` varchar(100) NOT NULL DEFAULT '',
+  `title` varchar(100) NOT NULL DEFAULT '',
+  `state` int(11) NOT NULL DEFAULT '0',
   `created_on` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
```

Comment:
@divyachaudhari @ankush-maherwal 

for dates instead of setting default as 0000-00-00 00:00:00, set column nullable with default value as NULL 

```
`created_on` datetime NULL DEFAULT NULL
```

refer https://github.com/joomla/joomla-cms/blob/0a1308f514328b3f4e56c8c04e4fc5727c669db7/installation/sql/mysql/extensions.sql#L168